USAGE.md

```diff
@@ -496,3 +498,106 @@ Get-GitHubEvent -OwnerName Microsoft -RepositoryName PowerShellForGitHub -Issue
 ```powershell
 Get-GitHubEvent -OwnerName Microsoft -RepositoryName PowerShellForGitHub -EventID 1
 ```
+
+----------
+
+### Advanced
+
+#### Migrating blog comments to github issues
+
+I used this module to migrate my blog comments from Disqus to Github Issues. I'm using Utterances App to display the Github Issues on each blog posts. [See this blog post for more details](https://lazywinadmin.com/2019/04/moving_blog_comments.html)
```

Comment:
Please use the third-person here (instead of saying "I", refer to yourself by name or GitHub username).  Similar feedback for any other pronouns.  The perspective of who wrote this content will be lost to the reader who is reading the documentation by itself vs from within the context of this PR.

Reply:
addressed in last updated